src/rtc/h5_prep.py

```diff
@@ -576,3 +802,109 @@ def save_hdf5_dataset(ds_filename, h5py_obj, root_path,
 
     del gdal_ds
 
+
+def populate_rfi_info(h5py_obj, burst, rfi_root_path):
+    '''
+    Populate the RFI information into HDF5 object
+
+    Parameters
+    ----------
+    h5py_obj: HDF5 object
+    burst: Sentinel1BurstSlc
+    rfi_root_path: str
+
+    '''
+    is_rfi_info_empty = burst.burst_rfi_info is None
+    dset = h5py_obj.create_dataset(f'{rfi_root_path}/isRfiInfoAvailable',
```

Comment:
I think we can benefit from following the ESA's way of using acronym in the field name, because we can extract the metadata from the annotation file easily.
If you don't have strong opinion about capitalizing `RFI`, then I would like to follow the ESA's convention.

Reply:
Another thought: I think we can use `RFI` in the fields that we've added, and use `'Rfi` if we extract the values from the annotation.

Reply:
Sounds good, @seongsujeong , I'm fine with having `Rfi`. I didn't quite understand the second thought.

Reply:
For example, the field name `isRFIInfoAvailable` above is not in the RFI annotation. I've added this field to explicitly indicate the existence of RFI information.
I thought we can use the capitalized acronym in that case.